README.md

```diff
@@ -3,55 +3,61 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/a3a966589be730bc865e/test_coverage)](https://codeclimate.com/github/openwell/questioner/test_coverage)
+# Crowd-source questions for a meetup.
 
-Crowd-source questions for a meetup. Questioner helps the meetup organizer prioritize
-questions to be answered. Other users can vote on asked questions and they bubble to the top
-or bottom of the log..
+Questioner helps the meetup organizer prioritize
+questions to be answered.
 
 ## Getting Started
 
 Clone the Repo.
 -------------
-Run npm install to install all necessary packages.
+Run `npm install` to install all necessary packages.
 
 ### Prerequisites
 
 The following tools will be needed to run this application successfully:
 
-Node v10.13.0 or above
+Node v8.11.3 or above
 ---
-Npm v6.4.1 or above
+Npm v5.6 or above
 ---
 
 ### ENDPOINTS
 
-- POST **/api/v1/meetups** Create an meetup record.
-- GET **/api/v1/meetups** Fetch all meetup records.
-- GET **/api/v1/meetups/1/:meetup-id** Fetch a specific meetup record.
-- GET **/api/v1/meetups/upcoming/** Fetch all upcoming meetup records.
-- POST **/api/v1/questions** Create a question for a specific meetup.
-- PATCH **/api/v1/questions/:question-id/upvote** Upvote (increase votes by 1) a specific question.
-- PATCH **/api/v1/questions/:question-id/downvote** Downvote (decrease votes by 1) a specific question.
-- POST *- POST **/api/v1/auth/signup** Create a user account.
-- POST **/api/v1/auth/login** Login a user
-- POST **/api/v1/auth/admin** Login as an admin
-- POST **/api/v1/comments/** Comment on a question.
-- GET **/api/v1/comments/:question-Id/** Get Comments of a Question.
-- DELETE **/api/v1/meetups/:meetup-id** Delete a meetup
+- POST `/api/v1/meetups` Create an meetup record.
+- GET `/api/v1/meetups` Fetch all meetup records.
+- GET `/api/v1/meetups/1/:meetup-id` Fetch a specific meetup record.
+- GET `/api/v1/meetups/upcoming/` Fetch all upcoming meetup records.
+- POST `/api/v1/questions` Create a question for a specific meetup.
+- GET `/api/v1/questions` Fetch all questions.
+- PATCH `/api/v1/questions/:question-id/upvote` Upvote (increase votes by 1) a specific question.
+- PATCH `/api/v1/questions/:question-id/downvote` Downvote (decrease votes by 1) a specific question.
+- POST `/api/v1/meetups/:meetup-id/rsvps` Respond to meetup RSVP.
+- POST `/api/v1/auth/signup` Create a user account.
+- POST `/api/v1/auth/login` Login a user
+- POST `/api/v1/auth/admin` Login as an admin
+- POST `/api/v1/comments/` Comment on a question.
+- GET `/api/v1/comments/:question-Id/` Comment on a question.
+- DELETE `/api/v1/meetups/:meetup-id` Delete a meetup.
+
 
 ### Installing
 
 ## On your Local Machine
-- Pull the [develop](https://github.com/openwell/questioner) branch off this repository
+- Pull the [develop](https://github.com/openwell/Questioner) branch off this repository
+- Install postgreSql 11 on your local machine or registered on any online platform
+- create a .env file
```

Comment:
Why not make this rename sample.env file to .env and input all the required environment variables.